flake.nix

```diff
@@ -0,0 +1,26 @@
+{
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs";
+    flake-utils.url = "github:numtide/flake-utils";
```

Comment:
I notice that you don't actually use the `flake-utils` input, so I'd take that out.